platforms/intellij/compose/playground/src/jvmTest/kotlin/foundry/intellij/compose/playground/MarkdownPlaygroundTest.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2024 Slack Technologies, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package foundry.intellij.compose.playground
+
+import androidx.compose.ui.test.ExperimentalTestApi
+import androidx.compose.ui.test.onNodeWithTag
+import androidx.compose.ui.test.onRoot
+import androidx.compose.ui.test.performClick
+import androidx.compose.ui.test.runDesktopComposeUiTest
+import com.github.takahirom.roborazzi.RoborazziOptions
+import io.github.takahirom.roborazzi.captureRoboImage
+import kotlinx.coroutines.test.runTest
+import org.junit.Test
+
+class MarkdownPlaygroundKtTest {
+  @OptIn(ExperimentalTestApi::class)
+  @Test
+  fun test() = runTest {
+    runDesktopComposeUiTest {
+      setContent { AppContent() }
+
+      val roborazziOptions =
```

Comment:
If you define Roborazzi options/rule globally like [this](https://github.com/takahirom/roborazzi/tree/1d5b51d55e4c2b3cfaca13cf6d38bebc0d1fae23?tab=readme-ov-file#image-comparator-custom-settings), it will be easy to add more tests and you don't need to pass `roborazziOptions` whenever you call `captureRoboImage()` API.

Reply:
Is `RoborazziRule` supposed to be provided by the Roborazzi plugin? It was asking me to create my own class for `RoborazziRule` when I tried to add it.

Reply:
yes, you need to add this. `testing-roborazzi-rules = { module = "io.github.takahirom.roborazzi:roborazzi-junit-rule", version.ref = "roborazzi" }`

Reply:
These are things we use in our slack-android-ng. you may need to see which additional ones are necessary.
```
testing-roborazzi = { module = "io.github.takahirom.roborazzi:roborazzi", version.ref = "roborazzi" }
testing-roborazzi-rules = { module = "io.github.takahirom.roborazzi:roborazzi-junit-rule", version.ref = "roborazzi" }
testing-roborazzi-compose = { module = "io.github.takahirom.roborazzi:roborazzi-compose", version.ref = "roborazzi" }
testing-roborazzi-core = { module = "io.github.takahirom.roborazzi:roborazzi-core", version.ref = "roborazzi" }
```